common/OpTestHMC.py

```diff
@@ -349,14 +349,27 @@ def get_proc_mode(self):
         return self.run_command("lshwres -r proc -m %s --level lpar --filter lpar_names=%s -F curr_proc_mode" %
                                (self.mg_system, self.lpar_name))
 
-    def enable_disable_vtpm(self, vtpm_mode):
+    def disable_vtpm(self):
         '''
-        Enables or disables vtpm mode.
+        disables vtpm mode.
 
-        :param vtpm_mode: Enables vtpm if vtpm_mode is 1 and disables vtpm if vtpm_mode is 0
         '''
-        self.run_command("chsyscfg -r lpar -m %s -i \"name=%s, vtpm_enabled=%s\"" %
-                               (self.mg_system, self.lpar_name, vtpm_mode))
+        self.run_command("chsyscfg -r lpar -m %s -i \"name=%s, vtpm_enabled=0\"" %
+                        (self.mg_system, self.lpar_name))
+        time.sleep(5)
+
+    def enable_vtpm(self, vtpm_version):
+        '''
+        Enables vtpm mode.
+
+        :param vtpm_version: Specifies vtpm version 1.2 or 2.0
+        '''
+        if vtpm_version == 2.0:
+            self.run_command("chsyscfg -r lpar -m %s -i \"name=%s, vtpm_enabled=1, vtpm_version=2.0, vtpm_encryption=Power10\"" %
```

Comment:
vtpm_encryption takes Power10v1 or Power10v2. At this moment we should go with `Power10v1`. Please change the above `Power10` to `Power10v1`.

Reply:
HMC with different 1050 PFW version are behaving differently. Leave it like this at this moment. I will get back to you as soon as I can. Thank You.